src/solver/utils/mps_utils.h

```diff
@@ -36,10 +36,11 @@ class I_MPS_writer
 class fullMPSwriter final : public I_MPS_writer
 {
 public:
-	fullMPSwriter(PROBLEME_SIMPLEXE_NOMME* named_splx_problem, uint thread_number);
+	fullMPSwriter(PROBLEME_SIMPLEXE_NOMME* named_splx_problem, int currentOptimNumber, uint thread_number);
```

Comment:
Some data members are common to all MPS writers but the null writer.
Do we create a class **MPSwriter** containing the common data members ?

Reply:
Not really required